saleor/graphql/product/filters.py

```diff
@@ -797,6 +810,182 @@ def filter_stock_availability(self, queryset, name, value):
         return _filter_stock_availability(queryset, name, value, channel_slug)
 
 
+class ProductWhere(MetadataFilterBase):
```

Comment:
A lot of fields don't have `help_text`. In my opinion, all new fields in API should have descriptions.